DataConnectors/WireXsystemsNFP(1b).json

```diff
@@ -104,7 +115,7 @@
                             "WorkspaceId"
                         ],
                         "label": "Run the following command to validate your connectivity:",
-                        "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"
+                        "value": "sudo wget -O cef_troubleshoot.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_troubleshoot.py&&sudo python cef_troubleshoot.py  {0}"
```

Comment:
@v-jayakal - Please do this change in the CEF template file as well https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json - approving this PR though.